engine/hid/src/native/hid_gamepad_driver_glfw.cpp

```diff
@@ -148,13 +183,17 @@ namespace dmHID
             {
                 return;
             }
+
             SetGamepadConnectionStatus(g_GLFWGamepadDriver->m_HidContext, gp, event == dmPlatform::GAMEPAD_EVENT_CONNECTED ? 1 : 0);
         }
     }
 
-    static inline uint8_t* RemapGamepadWin32(Gamepad* gamepad, uint8_t* buttons, uint8_t* buttons_remapped)
+    static inline uint8_t* RemapGamepadWin32(GLFWGamepadDevice* glfw_gamepad, uint8_t* buttons, uint8_t* buttons_remapped)

```

Comment:
Perhaps ifdef the function as well, to make sure it's not accidentally included in another platform (even if it's called)